draft-mouris-cfrg-mastic.md

```diff
@@ -661,15 +680,154 @@ malicious Aggregator.
 
 > NOTE to be specified in full detail.
 
+# Applications of `Mastic`
+
+Based on the two modes of operation of Mastic (i.e., Weighted Heavy-Hitters
+{{weighted-heavy-hitters}} and Attribute-Based Metrics
+{{aggregation-by-labels}}), we describe two applications of interest.
+
+## Network Error Logging {#NEL}
+Network Error Logging (NEL) is a mechanism used by web browsers to report errors
+that occur while attempting to establish a connection to a server {{W3C23}}. Some
+of these errors are visible to the server, but not all: failures in DNS, TCP,
+TLS, and HTTP can occur without the server having any visibility into the issue.
+A small amount of connection errors is expected, even under normal operating
+conditions; but a sudden, substantial increase in errors may be an indication of
+an outage, or a configuration issue impacting millions of users. Without a
+reporting mechanism like NEL, these events would only manifest in the server's
+telemetry as a drop in overall traffic.
+
+NEL is particularly important for content delivery networks, such as Akamai,
+AWS, or Cloudflare, that handle HTTP traffic for a large number of websites
```

Comment:
Internet-Drafts shouldn't reference specific companies or institutions. "Content delivery network" is clear enough to the intended audience.

```suggestion
NEL is particularly important for content delivery networks that handle HTTP traffic for a large number of websites
```